docs/platform_developers/developer_getting_started.rst

```diff
@@ -0,0 +1,211 @@
+2. Working on an existing project :ref:`Working on an Existing Project`.
+
+
+Working on a New Project
+========================
+
+To create a new project: 
+
+.. code-block:: bash
+
+    pip install cookiecutter
+
+    cookiecutter https://github.com/inmanta/inmanta-project-template.git
+
+
+For more details go `here <https://docs.inmanta.com/community/latest/model_developers/configurationmodel.html>`_.
+
+
+You need to install some essential packages as follows:
+
+.. code-block:: bash
+
+    pip install inmanta-core
+
+    pip install pytest
+
+    pip install pytest-inmanta
+
+
+Once you are done with creating a project and installing the required modules, you can ``cd`` into that directory and open vs code by running:
+
+.. code-block:: bash
+    
+    cd <project_name>
+    
+    code .
+
+Upon opening your vs code, and the ``main.cf`` file, you should see modules downloading in ``libs`` directory.
+
+
+Working on an Existing Project
+==============================
+
+When working on an existing project, you need to ``clone`` them first:
+
+.. code-block:: bash
+
+    git clone project_name
+
+
+They also come with ``requirements.txt`` or ``requirements.dev.txt`` to install the required modules:
+
+.. code-block:: bash
+
+    pip install -r requirements.txt
+
+    pip install -r requirements.dev.txt
+
+
+Set project sources
+#####################
+
+When starting a new project, the next step is to set the sources of your project so that it knows, where to get its required modules from. Otherwise, you can skip this step and just ``import`` your desired modules.
+
+If you only use opensource modules as provided by Inmanta, you can skip below step. 
+
+1. Find the module you want to work on
+2. Copy the SSH URL of the repo
+3. In your VS code, open the ``project.yml`` file and under ``repo:``, add the copied line there but keep in mind to replace the name of a specific module with a place holder, like below example:
+
+.. code-block:: bash
+
+    code project.yml
+
+.. code-block:: yaml
+    
+    repo:
+        - git@code.inmanta.com:example/my_module.git
+
+Becomes:
+
+.. code-block:: yaml
+    
+    repo:
+        - git@code.inmanta.com:example/{}.git
+
+* Now, in your ``main.cf`` file, if you import a module like, ``import <my_module>`` and save the file, you can get code completion. If you are working on an exisitng project with a populated ``main.``cf file, code completion will work as expected.
+
+**Please note, code completion and navigation work on modules that are imported in the ``main.cf`` file**.
+
+
+Module developers guide
+#########################
+
+Like projects, there are also two scenarios:
+
+1. Working on a new module :ref:`Working on a New Module`.
+2. Working on an existing module :ref:`Working on an Existing Module`.
+
+
+Working on a New Module
+=======================
+
+There is a guide `here <https://docs.inmanta.com/community/latest/model_developers/modules.html>`_ that helps you get up and running.
```

Comment:
you refer to this guide above and below the example, I would structure this consistent to the new projects part.